pkg/services/ngalert/ngalert.go

```diff
@@ -487,13 +487,13 @@ func (ng *AlertNG) init() error {
 }
 
 func subscribeToFolderChanges(logger log.Logger, bus bus.Bus, dbStore api.RuleStore) {
-	// if folder title is changed, we update all alert rules in that folder to make sure that all peers (in HA mode) will update folder title and
+	// if full path to the folder is changed, we update all alert rules in that folder to make sure that all peers (in HA mode) will update folder title and
 	// clean up the current state
-	bus.AddEventListener(func(ctx context.Context, evt *events.FolderTitleUpdated) error {
-		logger.Info("Got folder title updated event. updating rules in the folder", "folderUID", evt.UID)
-		_, err := dbStore.IncreaseVersionForAllRulesInNamespace(ctx, evt.OrgID, evt.UID)
+	bus.AddEventListener(func(ctx context.Context, evt *events.FolderFullPathUpdated) error {
+		logger.Info("Got folder full path updated event. updating rules in the folders", "folderUIDs", evt.UIDs)
+		_, err := dbStore.IncreaseVersionForAllRulesInNamespaces(ctx, evt.OrgID, evt.UIDs)
```

Comment:
I wonder if we want to log keys of updated rules here.

Reply:
I was thinking that we could completely remove fetching the keys in `IncreaseVersionForAllRulesInNamespaces` since we are not using them. Plus, it doesn't return updated IDs but makes another query to fetch UIDs, which is potentially inconsistent. What do you think?

Reply:
It would be great if we can log the UID of changed rules so we have a full picture of what happened.